src/lib/utils/robotUtils.ts

```diff
@@ -0,0 +1,3 @@
+export function isOnTheBoard(num: number): boolean {
+  return num >= 0 && num <= 4;
```

Comment:
I reckon it would be nice to make this more generic somehow, so we can spin up robots on grids of arbitrary size

Reply:
Sure, changed it 👍